src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -29,10 +29,17 @@ function exit_handler()
     "[DEBUG]" "Docker container exit handler: EXIT signal received in docker container, exiting ..."
 }
 
+function kill_handler()
+{
+  printf "%s %s\n" \
+    "[DEBUG]" "Docker container kill handler: SIGTERM signal received in docker container, exiting ..."
+    exit 0
+}
+
 set -x
 PS4="+[\t] "
 trap exit_handler EXIT
-
+trap kill_handler SIGTERM
```

Comment:
`docker kill` support `-s` option to choose signal, please trap other signal instead of SIGTERM, which is used in many places, such as https://github.com/Microsoft/pai/blob/acb982d1272924dffa49d5cc220d5f1bb6a40594/src/hadoop-node-manager/deploy/hadoop-node-manager-delete/delete.sh#L27

Reply:
Accepted